ttnn/cpp/ttnn/operations/matmul/device/matmul_op.cpp

```diff
@@ -811,6 +811,20 @@ inline MatmulProgramConfig get_program_config(
     return config;
 }
 
+tt::tt_metal::Tile get_output_tile(const MemoryConfig& output_mem_config, const std::array<uint32_t, 2>& in0_tile_shape, const std::array<uint32_t, 2>& in1_tile_shape, const std::optional<const tt::tt_metal::Tile> output_tile) {
+    if (output_tile.has_value()) {
+        TT_FATAL(output_tile->get_tile_shape()[1] % in1_tile_shape[1] == 0, "the override output tile width be multiple of in1 tile width");
```

Comment:
is index 1 width and index 0 height ?

Reply:
yes, index 1 is width and index 0 is height:
tt_metal/impl/tile/tile.hpp
```
    std::array<uint32_t, 2> tile_shape = {constants::TILE_HEIGHT, constants::TILE_WIDTH};
```

I added separate checks that the value is greater than 0.